cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/sdn/YFMan/High-Precision-Congestion-Control-master/simulation/.waf-1.7.11-edc6ccb516c5e3f9b892efc9f53a610f/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/home/sdn/YFMan/High-Precision-Congestion-Control-master/simulation/.waf-1.7.11-edc6ccb516c5e3f9b892efc9f53a610f/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140284694879184: cxx test.cpp -> test.cpp.1.o}\n[\'g++-5\', \'-O0\', \'-ggdb\', \'-g3\', \'-std=gnu++11\', \'-DNS3_ASSERT_ENABLE\', \'-DNS3_LOG_ENABLE\', \'-DHAVE___UINT128_T=1\', \'-DINT64X64_USE_128=1\', \'-DHAVE_SYS_TYPES_H=1\', \'-DHAVE_SYS_STAT_H=1\', \'-DHAVE_DIRENT_H=1\', \'-DHAVE_STDLIB_H=1\', \'-DHAVE_GETENV=1\', \'-DHAVE_SIGNAL_H=1\', \'-DHAVE_PTHREAD_H=1\', \'-DHAVE_STDINT_H=1\', \'-DHAVE_INTTYPES_H=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
